plan error::args {
  fail_plan('oops', 'test/oops', { 'some' => 'info' })
}
